### DISCUSSION TITLE
[Request] ETA on external regressors in hierarchical forecasts

### DISCUSSION THREAD

**Discussion by jacmag** [microsoft/finnts#105] [Q&A] [answered]
Hello,
Love the work you've done on this package. Do you have an ETA on adding external regressors to hierarchical forecasts?
Thank you!

**Comment by mitokic** [comment, marked as answer]
We don't have an ETA on the feature, but we are currently working on it. We're planning on moving away from the hts package and would need to build a lot of things from the ground up. Things like the initial aggregation process and then the final reconciliation process need to be custom built. This will take time.

**Reply by jacmag** [reply]
Thanks for the update. Again, really appreciate all the work.
